core/service/src/lib.rs

```diff
@@ -159,11 +158,11 @@ impl<Components: components::Components> Service<Components> {
 			select_chain.clone(),
 		)?);
 		let finality_proof_provider = Components::build_finality_proof_provider(client.clone())?;
-		let best_header = select_chain.best_chain()?;
+		let chain_info = client.info()?.chain;
 
 		let version = config.full_version();
-		info!("Best block: #{}", best_header.number());
-		telemetry!(SUBSTRATE_INFO; "node.start"; "height" => best_header.number().as_(), "best" => ?best_header.hash());
+		info!("Best block: #{}", chain_info.best_number);
```

Comment:
I suggest to change this message. "Best" is only correct in the context of a longest-chain strategy (and thinking), which isn't necessarily the case anymore. But with a slight rewording this is clear imho:

```suggestion
		info!("Highest known block at #{}", chain_info.best_number);
```